scheduler/src/cook/kubernetes/controller.clj

```diff
@@ -498,19 +499,18 @@
                                               ; sure to write the status to datomic. Recall we're in kubernetes state missing.
                                               (handle-pod-killed compute-cluster pod-name))
                                             (do
-                                              ; We treat a deleting pod in kubernetes the same as a missing pod when coming up with a synthesized state.
-                                              ; That's good for (almost) all parts of the system. However,
-                                              ; If it is legitimately missing, then something weird is going on. If it is
-                                              ; deleting, that's an expected state. So, let's be selective with our logging.
-                                              (if (= (:state synthesized-state) :missing)
-                                                (log/info "In compute cluster" name ", pod" pod-name
-                                                          "was killed with cook expected state"
-                                                          (prepare-cook-expected-state-dict-for-logging cook-expected-state-dict)
-                                                          "and k8s actual state"
-                                                          (prepare-k8s-actual-state-dict-for-logging k8s-actual-state-dict))
-                                                (log-weird-state compute-cluster pod-name
-                                                                 cook-expected-state-dict k8s-actual-state-dict))
+                                              (log-weird-state compute-cluster pod-name
```

Comment:
May be worth a comment that this shouldn't occur except in the cases of extreme staleness; if you miss the deleting intermediate states.

Reply:
Done